indico/modules/events/abstracts/templates/abstract/judge.html

```diff
@@ -37,43 +37,97 @@
-    <span class="i-tag outline {{ mapping_class[state.name] }}" title="{{ mapping_title[state.name] }}">
-        {{ state.title -}}
-    </span>
+    <div class="state-icon {{ mapping_class[state.name] }} flexrow f-a-center f-j-center"
+         title="{{ mapping_title[state.name] }}">
+        <span
+            {%- if state.name in ['positive', 'conflicting'] %}
+                class="icon-checkmark"
+            {% elif state.name in ['mixed', 'negative'] %}
+                class="icon-cross"
+            {%- endif -%}
+        ></span>
+    </div>
 {% endmacro %}
 
-
 {% macro render_decision_box(abstract, form, management=false) %}
     <div class="i-timeline-item-label action icon-hammer"></div>
     <div class="i-timeline-item-box content-indicator">
-        <div class="i-box-content">
-            <div class="flex">
+        <div class="i-box-content track-reviews-wrapper">
+            <div class="tracks-review-summary flexrow f-a-center">
                 <div class="stretch">
                     <strong>
-                        {% trans %}Review summary{% endtrans %}
+                        {%- trans %}Review summary{% endtrans -%}
                     </strong>
                 </div>
-                <div class="abstract-comment-badges">
-                    {% trans state=_render_abstract_reviewing_state(abstract) -%}
-                        Reviews are {{ state }}
-                    {%- endtrans %}
+                <div class="abstract-comment-badges flexrow f-a-center">
+                    <span class="review-state-pretext">{% trans %}Reviews are{% endtrans %}</span>
+                    {{ _render_abstract_reviewing_state(abstract) }}
                     {% if abstract.score is not none %}
                         {{ render_score(abstract.score, abstract.event_new.cfa) }}
                     {% endif %}
                 </div>
             </div>
+            {% set ratings = abstract.get_avg_per_question_per_track() %}
             {% for track in abstract.reviewed_for_tracks|sort(attribute='title') %}
-                <div class="track-review flex">
-                    <div class="stretch">
-                        {{ track.title }}
-                    </div>
-                    <div>
-                        {{ _render_abstract_track_reviewing_state(abstract, track) }}
+                <div class="track-review">
+                    <div class="track-review-summary flexrow f-a-center">
+                        {{ _render_abstract_track_reviewing_state_icon(abstract, track) }}
+                        <div class="stretch">
+                            {{- track.title -}}
+                        </div>
                         {% set score = abstract.get_track_score(track) %}
+                        {% if abstract.reviews %}
+                            <a href="#" class="track-review-details-link js-details-hidden js-toggle-details"
+                               data-target="review-list"
```

Comment:
`.review-list` and remove the `.` from the JS file - currently you have no idea if that's a class or ID when seeing this.